recipes/chemps2/build.sh

```diff
@@ -0,0 +1,71 @@
+# set -x
+
+LAPACK_INTERJECT="${PREFIX}/lib/libmkl_rt.so"
+# link against conda MKL & GCC
+# if [ "$blas_impl" = "mkl" ]; then
+#     LAPACK_INTERJECT="${PREFIX}/lib/libmkl_rt.so"
+# else
+#     LAPACK_INTERJECT="${PREFIX}/lib/libopenblas.so"
+# fi
+# ALLOPTS="-gnu-prefix=${HOST}- ${OPTS}"
+
+# configure
+${PREFIX}/bin/cmake \
```

Comment:
https://github.com/psi4/psi4meta/blob/master/conda-recipes/chemps2-multiout/build.sh#L13 since cmake is in build, not host. Or I think you can just call `cmake` since both build and host prefixes are activated.